diff --git a/hw/virtio/virtio-mmio.c b/hw/virtio/virtio-mmio.c
index 9d3db3c5b0..1b3c5e4f5a 100644
--- a/hw/virtio/virtio-mmio.c
+++ b/hw/virtio/virtio-mmio.c
@@ -769,7 +769,7 @@ static const Property virtio_mmio_properties[] = {
-    DEFINE_PROP_BOOL("force-legacy", VirtIOMMIOProxy, legacy, true),
+    DEFINE_PROP_BOOL("force-legacy", VirtIOMMIOProxy, legacy, false),
     DEFINE_PROP_BOOL("format_transport_address", VirtIOMMIOProxy,
                      format_transport_address, true),
     DEFINE_PROP_BOOL("ioeventfd", VirtIOMMIOProxy, ioeventfd, true),
     DEFINE_PROP_END_OF_LIST(),
 };
